build.sbt

```diff
@@ -32,7 +42,6 @@ lazy val compilerFlags = Seq(
     "-unchecked",
     "-Xfuture",
```

Comment:
Looks like this flag is also deprecated.

Reply:
Removed.